R/kelp.R

```diff
@@ -0,0 +1,110 @@
+#' Create a kelp object for interacting with SeaweedFS.
+#'
+#' This is a slightly higher level abstraction than [seaweed_master()] and
+#' [seaweed_volume()].
+#'
+#' @keywords export
```

Comment:
You might add a small doc note here about the difference between object and file, and what happens if you did `download_object` on something that was saved as an object vs saved as a file

Reply:
I've made that documentation a bit more clear, let me know if still needs to be tidied up at all? Also updated to now error if you try to `download_object` on something which it cannot unserialize.